packages/terra-core-docs/src/terra-dev-site/doc/image/DecorativeImage.2.doc.mdx

```diff
@@ -0,0 +1,61 @@
+import { Badge } from 'terra-image/package.json?dev-site-package';
+
+import DecorativeImageVaraints from './example/DecorativeImage/DecorativeImageVaraints?dev-site-example';
+import DecorativeImageFit from './example/DecorativeImage/DecorativeImageFitTypes?dev-site-example';
+import DecorativeImageBar from './example/DecorativeImage/DecorativeImageBar?dev-site-example';
+import DecorativeImageTextLink from './example/DecorativeImage/DecorativeImageTextLink?dev-site-example';
+
+import DecorativeImagePropsTable from 'terra-image/src/variants/_DecorativeImage?dev-site-props-table';
+
+<Badge />
+
+# Decorative Image
+
+Decorative images don’t add information to the content of a page. For example, the information provided by the image might already be given using adjacent text, or the image might be included to make the website more visually attractive.
+
+Decorative Image should be used only when the purpose of an image is to add visual interest to the page, rather than to convey information that is important to understanding the page.
+
+Images may be decorative when they are:
+- Visual styling such as borders, spacers, and corners;
+- Supplementary to link text to improve its appearance or increase the clickable area;
+- Illustrative of adjacent text but not contributing information (“eye-candy”);
+- Identified and described by surrounding text.
```

Comment:
Nit: not sure why semi-colons are used here.
```suggestion
Images may be decorative when they are:
- Visual styling such as borders, spacers, and corners.
- Supplementary to link text to improve its appearance or increase the clickable area.
- Illustrative of adjacent text but not contributing information (“eye-candy”).
- Identified and described by surrounding text.
```

Reply:
[updated](https://github.com/cerner/terra-core/pull/3585/commits/a72252e81ed3e48fbebc3ff03878cf6f428b0261#diff-d0b823c53af1996652425cf428ede93a6a3f54e35ce19732897a0375f26b9c62)